lerna run swallows errors

this is similar to #119 but larger: the issue seems to include other commands as well.

`lerna run task` only gives me the exit code and swallows all logging the task may have given, which usually includes the compiler error message.

here's the output from `lerna tun ts`:

```
Lerna v2.0.0-beta.13
Independent Versioning Mode
Errored while running npm script 'ts' in '@elements/blueprint'
Error: Command failed: /bin/sh -c npm run ts

npm ERR! Darwin 15.4.0
npm ERR! argv "/usr/local/Cellar/node/5.3.0/bin/node" "/usr/local/bin/npm" "run" "ts"
npm ERR! node v5.3.0
npm ERR! npm  v3.3.12
npm ERR! code ELIFECYCLE
npm ERR! @elements/blueprint@2.29.0 ts: `tsc --project .`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the @elements/blueprint@2.29.0 ts script 'tsc --project .'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the @elements/blueprint package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     tsc --project .
npm ERR! You can get their info via:
npm ERR!     npm owner ls @elements/blueprint
npm ERR! There is likely additional logging output above.
```

and here's the output from running the first lerna task directly in its project directory:

```
> @elements/blueprint@2.29.0 ts /Users/ggray/palantir/blueprint/packages/blueprint
> tsc --project .

error TS6053: File 'src/generated/iconClasses.ts' not found.
error TS6053: File 'src/generated/iconStrings.ts' not found.

npm ERR! Darwin 15.4.0
npm ERR! argv "/usr/local/Cellar/node/5.3.0/bin/node" "/usr/local/bin/npm" "run" "ts"
npm ERR! node v5.3.0
npm ERR! npm  v3.3.12
npm ERR! code ELIFECYCLE
npm ERR! @elements/blueprint@2.29.0 ts: `tsc --project .`
npm ERR! Exit status 2
```

i've noticed other similar discrepancies between the logging during a `lerna run *` versus during a direct `npm run *` in the project, where lerna gives much less information. perhaps there's an extended buffer on the logger?

--- a/src/ChildProcessUtilities.js
+++ b/src/ChildProcessUtilities.js
@@ -14,7 +14,7 @@ export default class ChildProcessUtilities {
           err = `Error: ${err.message}.  Partial output follows:\n\n${stderr}`;
         }
 
-        callback(err || stderr);
+        callback(err || stderr, stdout);
       } else {
         callback(null, stdout);
       }

--- a/src/commands/RunCommand.js
+++ b/src/commands/RunCommand.js
@@ -54,10 +54,9 @@ export default class RunCommand extends Command {
 
   runScriptInPackage(pkg, callback) {
     NpmUtilities.runScriptInDir(this.script, this.args, pkg.location, (err, stdout) => {
+      this.logger.info(stdout);
       if (err) {
         this.logger.error(`Errored while running npm script '${this.script}' in '${pkg.name}'`, err);
-      } else {
-        this.logger.info(stdout);
       }
       callback(err);
     });

